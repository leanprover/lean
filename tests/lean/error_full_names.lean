namespace foo
open nat
inductive nat : Type | zero, foosucc : nat â†’ nat
check 0 + nat.zero --error
end foo

namespace foo
check nat.succ nat.zero --error
end foo
