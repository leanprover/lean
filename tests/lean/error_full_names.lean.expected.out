error_full_names.lean:4:8: error: failed to synthesize placeholder

⊢ has_add nat
error_full_names.lean:8:6: error: type mismatch at application
  nat.succ nat.zero
term
  nat.zero
has type
  nat
but is expected to have type
  ℕ
